examples/clinical_pipeline/README.md

```diff
@@ -18,7 +18,7 @@ named entities corresponding to diseases, symptoms and related concepts.
 Before running the pipeline, we need to download the models
 
 ```bash
-python ../biobert_ner/download_models.py 
+python ../../fortex/huggingface/biobert_ner/download_models.py 
```

Comment:
Users may not have the `fortex` folder in their environment. Maybe it is easier to copy the script into this example